b2b-sample/README.md

```diff
@@ -47,6 +47,8 @@ Now we need **two** applications to communicate with the **Guardio-Business-App*
 
 Create two applications named **Guardio-Business-App** and **Guardio-Admin-App** [This should be a management application].
 
+Share both applications with all the organizations by clicking the share button.
+
```

Comment:
Why Guardio-Business-App needs a client credential grant?
I think it should be changed to  `Organization Switch` and `Code`

Reply:
Yes, We can skip `client credential` grant
Fixed by - https://github.com/wso2/samples-is/pull/405/commits/221eee7cb4d209292210fbf3be7894dc667b7249